### PR TITLE
Clean up typos and run-on sentences.

### PR DESCRIPTION
**Pull request by SSteve**

### CODE REVIEW

File: docs/chapters/abc/content.en-GB.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ De Casteljau's algorithm is the pivotal algorithm when it comes to Bézier curve
...
 
-So let's use graphics instead of text to see where this "A" is, because text only gets us so far: move the sliders for the following graphics to see what, given specific `t` value, our `A` coordinate is. As well as some other coordinates, which taken together let us derive a value that the graphics call "ratio": if you move the curve's points around, A, B, and C will move, what happens to that value?
+So let's use graphics instead of text to see where this "A" is, because text only gets us so far: move the sliders for the following graphics to see what, given a specific `t` value, our `A` coordinate is. As well as some other coordinates, which taken together let us derive a value that the graphics call "ratio": if you move the curve's points around, A, B, and C will move. What happens to that value?
```
Comment (Pomax) [inline docs/chapters/abc/content.en-GB.md:9]: this comma cannot become a full stop, the subphrase is "if you move the curve's points around, A, B, and C will move, what happens to that value?"

File: docs/chapters/abc/content.en-GB.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ And
   u(t)_{cubic} = \frac{(1-t)^3}{t^3 + (1-t)^3}
 \]
 
-So, if we know the start and end coordinates, and we know the *t* value, we know C, without having to calculate the `A` or even `B` coordinates. In fact, we can do the same for the ratio function: as another function of `t`, we technically don't need to know what `A` or `B` or `C` are, we can express it was a pure function of `t`, too.
+So, if we know the start and end coordinates and the *t* value, we know C without having to calculate the `A` or even `B` coordinates. In fact, we can do the same for the ratio function. As another function of `t`, we technically don't need to know what `A` or `B` or `C` are. We can express it too as a pure function of `t`.
```
Comment (Pomax) [inline docs/chapters/abc/content.en-GB.md:50]: If you want to include a "too", then it'll have to become "It, too, can be expressed as a pure function of `t`."

File: docs/chapters/abc/content.en-GB.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ And
   ratio(t)_{cubic} = \left | \frac{t^3 + (1-t)^3 - 1}{t^3 + (1-t)^3} \right |
 \]
 
-Which now leaves us with some powerful tools: given thee points (start, end, and "some point on the curve"), as well as a `t` value, we can _construct_ curves: we can compute `C` using the start and end points, and our `u(t)` function, and once we have `C`, we can use our on-curve point (`B`) and the `ratio(t)` function to find `A`:
+Which now leaves us with some powerful tools: given three points (start, end, and "some point on the curve"), as well as a `t` value, we can _construct_ curves. We can compute `C` using the start and end points and our `u(t)` function. Once we have `C`, we can use our on-curve point (`B`) and the `ratio(t)` function to find `A`:
```
Comment (Pomax) [inline docs/chapters/abc/content.en-GB.md:70]: I'd like to keep this "and".

**Review by Pomax — CHANGES_REQUESTED** [review]

File: docs/chapters/abc/content.en-GB.md (hunk 2 above)
Comment (SSteve) [reply docs/chapters/abc/content.en-GB.md:50]: Much better.

File: docs/chapters/abc/content.en-GB.md (hunk 1 above)
Comment (SSteve) [reply docs/chapters/abc/content.en-GB.md]: Ok.

File: docs/chapters/abc/content.en-GB.md (hunk 3 above)
Comment (SSteve) [reply docs/chapters/abc/content.en-GB.md]: Ok.

**Review by SSteve — COMMENTED** [review]
I made the requested changes in another commit. (I hope that's the correct way to do it. I haven't made changes based on conversations before.)

**Review by Pomax — APPROVED** [review]

**Comment by Pomax** [comment]
yep, that's exactly the right way to update PRs: each change ends up being a commit that becomes part of the PR, which means you don't have to close a PR and then open a new one, you just keep updating the files with new commits until the PR's good to go - which is right now =)

Thank you for the corrections!

**Pull request merged**